File: Basic Question Bank/a006/a006.py
coef_list = [eval(x) for x in input().split(' ')]
a = coef_list[0]; b = coef_list[1]; c = coef_list[2]
if b ** 2 - 4 * a * c < 0 :
    print("No real root")
elif b ** 2 - 4 * a * c == 0:
    print("Two same roots x=" + str(int((-b) / (2 * a))))
else:
    print("Two different roots x1=" + str(int(((-b) + ((b ** 2 - 4 * a * c) ** 0.5)) / (2 * a))) + " , x2=" + str(int(((-b) - ((b ** 2 - 4 * a * c) ** 0.5)) / (2 * a))))

File: Basic Question Bank/a006/a006.java
import java.util.Scanner;

public class a006 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt(), b = sc.nextInt(), c = sc.nextInt();
        int D = ((int)Math.pow(b, 2)) - 4 * a * c;
        if(D < 0) {
            System.out.println("No real root");
        }
        else if(D == 0) {
            System.out.println("Two same roots x=" + -b / (2 * a));
        }
        else {
            System.out.println("Two different roots x1=" + (-b + (int)Math.sqrt(D)) / (2 * a) + " , x2=" + (-b - (int)Math.sqrt(D)) / (2 * a));
        }
        sc.close();
    }
}
